src/arrayMethodPop.js

```diff
@@ -6,7 +6,13 @@
  */
 function applyCustomPop() {
   [].__proto__.pop2 = function() {
-    // write code here
+    let a = this[this.length - 1];
```

Comment:
Please avoid single-letter names
Probably you can move this code to `else` block along with `return a`. This way the code readability will increase.